src/components/Filter/filter.jsx

```diff
@@ -0,0 +1,54 @@
+import {useState} from "react"
+
+const Filter = ({ search }) => {
+  const [ displayDropdown, setDisplayDropdown ] = useState(false);
+
+  const searchValue = (e) => {
+    search(e.target.value); 
+  };
+
+  const regionValue = (region) => {
+    search(region);
+    setDisplayDropdown(false);
+  }
+
+  const dropdowntoggle = () => {
+    setDisplayDropdown(!displayDropdown);
+  };
+  
+  return (
+    <section className="filters">
+      <div className="container">
+        <div className="search-wrapper">
+          <i className="fa-regular fa-magnifying-glass search-icon"></i>
+          <input
+            type="text"
+            className="search-input"
+            placeholder="Search for a country..."
+            onChange={searchValue}
+          />
```

Comment:
If you call a function that only calling another funcntion maybe just call `searchValue` directly in here